iml-agent/src/action_plugins/ltuer.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use crate::agent_error::ImlAgentError;
+use std::path::PathBuf;
+use tokio_fs as fs;
+
+#[cfg(not(test))]
+static CONFIGURATION_DIR: &str = "/etc/iml";
+#[cfg(test)]
+static CONFIGURATION_DIR: &str = "/tmp/etc/iml";
+
+pub async fn create_ltuer_conf(
+    (mailbox_path, fs_name, cold_pool): (String, String, String),
+) -> Result<(), ImlAgentError> {
```

Comment:
It would be nice to describe the reason why the signature this peculiar? (Probably the reason is that some JSON should be convertible to the arguments, right?)

Reply:
Basically it's adhering to the plugin interface that takes a unary fn where the param is deserializable by serde and the result is serializable by serde.

ref: 

https://github.com/whamcloud/integrated-manager-for-lustre/blob/a3c1ed56df8d321709f36ac401e7115bb6a5bfa1/iml-util/util.rs#L137-L143

and:

https://github.com/whamcloud/integrated-manager-for-lustre/blob/a3c1ed56df8d321709f36ac401e7115bb6a5bfa1/iml-agent/src/action_plugins/action_plugin.rs#L20-L53

Reply:
Resolving as it doesn't have to do with this plugin itself